WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -428,7 +428,7 @@ private async Task<(bool success, string error)> TryRefreshHardwareWalletInfoAsy
 					}
 					var result = await pinpad.ShowDialogAsync();
 					DisplayActionTab();
-					if (!(result is true))
+					if (result is false)
```

Comment:
It's not the same. The result can be null and a cow, too. Please revert this change.